docs/development/oidc.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Testing OIDC
+weight: 5
+---
+
+OIDC is a protocol that allows web applications to authenticate users through an identity provider and obtain basic profile information about the user.
+
+## Testing OIDC with Kubernetes API Server
+
+This guide will walk you through the process of testing OIDC (OpenID Connect) on a Minikube cluster using an Azure App Registration. OIDC allows you to authenticate and interact with the cluster using kubectl:
+
+### Create an Application on Azure
+
+- Create an application on Azure and specify the redirect URL as `http://localhost:8000`.
```

Comment:
It'd be great to add links to Azure's docs every time we refer to something where there are docs about if they help users to more easily understand what to do. A link to the docs on creating this application would be very useful.